src/devRunner.js

```diff
@@ -0,0 +1,73 @@
+import webpack from 'webpack';
+import { resolve } from 'path';
+import fs from 'fs';
+import spawn from 'cross-spawn';
+
+const getCommonJsModules = path =>
+  fs
+    .readdirSync(path)
+    .filter(module => ['.bin'].indexOf(module) === -1)
+    .reduce((memo, module) => ({
+      ...memo,
+      [module]: `commonjs ${module}`
+    }), {});
+
+const SERVER_BUNDLE_NAME = 'bundle.server.js';
+
+export default (appRoot) => {
+  const resolveAppPath = path => resolve(appRoot, path);
+  const resolveLocalPath = path => resolve(__dirname, path);
+  const resolveLocalNodeDep = dependency => require.resolve(dependency);
+
+  const commonJsModules = getCommonJsModules(resolveAppPath('node_modules'));
+
+  const compiler = webpack({
+    entry: [
+      `${resolveLocalNodeDep('webpack/hot/poll')}?1000`,
+      resolveLocalPath('devServer.js')
+    ],
+    target: 'node',
+    externals: commonJsModules,
+    output: {
+      path: resolveLocalPath('../dist'),
+      filename: SERVER_BUNDLE_NAME
+    },
+    resolve: {
+      modules: [
```

Comment:
not sure if it's good idea tho, because `react` should obviously be installed in the application, so therefore where does it resolve from? App or `react-devstack`?